app/src/test/java/org/fossasia/openevent/app/core/presenter/LoginViewModelTest.java

```diff
@@ -0,0 +1,145 @@
+package org.fossasia.openevent.app.core.presenter;
```

Comment:
Wrong package. Change to as discussed on gitter

Reply:
Just to confirm, the package will be app.core.auth.login as it hs the vm